helpers/helpers.go

```diff
@@ -237,3 +238,22 @@ func Git(args ...string) {
 		os.Exit(1)
 	}
 }
+
+// RunCommand runs the given command with args and optionally prints
+func RunCommand(print bool, cmdname string, args ...string) error {
+	cmd := exec.Command(cmdname, args...)
+	if print {
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+	} else {
+		// Automatically connected to os.DevNull by implementation
+		cmd.Stdout = nil
+		cmd.Stderr = nil
```

Comment:
These are the default values for the exec.Cmd struct returned from exec.Command. So you can likely remove the else branch.

Reply:
Ah I knew this from looking at another PR of yours long time ago... ty for pointing this out again!